src/server/scripts/Northrend/Ulduar/Ulduar/boss_kologarn.cpp

```diff
@@ -124,6 +125,8 @@ enum Misc
     DATA_KOLOGARN_ARMS_ACHIEV           = 57,
 };
 
+Unit* eyebeamTarget;
+
```

Comment:
I don't think its a good idea to store target in pointer, maybe store to private variable inside of Kologard (uint64) to store its guid and attempt to retrieve target with ObjectAccessor

Reply:
Unfortunately this is used in both Kologarn AI and the Eye Beam NPC AI